pkg/s3select/select.go

```diff
@@ -470,13 +470,15 @@ func (s3Select *S3Select) Evaluate(w http.ResponseWriter) {
 				outputRecord = s3Select.outputRecord()
 				outputQueue[len(outputQueue)-1] = outputRecord
 			}
-			if err = s3Select.statement.Eval(inputRecord, outputRecord); err != nil {
```

Comment:
@klauspost Can these select related changes be in a separate PR? To ease the reviewing work?

Reply:
Whoops. Seems like an accidental commit. Great you spotted that.